tbx/project_styleguide/templates/patterns/atoms/see-more/see-more.html

```diff
@@ -1,11 +1,11 @@
 {% if text %}
     {% if fetch_data %}
```

Comment:
I think we can totally ditch this `fetch_data` logic - I think it was related to the js load more functionality

Reply:
Removed in https://github.com/torchbox/torchbox.com/pull/41/commits/9a5fc55cb8a3ea40c37cd123f9947d8b1d83e1f7